operations/mimir-mixin/dashboards/reads.libsonnet

```diff
@@ -249,6 +249,10 @@ local filename = 'mimir-reads.json';
         )
       )
     )
+    .addRowIf(
```

Comment:
Move it after the gateway autoscaling row, please

Reply:
Done.